### ISSUE TITLE
Project contains a bunch of "admit" commands

### ISSUE THREAD

**Issue by anton-trunov** [coq-community/lemma-overloading#5]
We should get rid of those.

**Comment by anton-trunov** [comment]
Partially solved by #13

**Comment by palmskog** [comment]
The only admitted lemma that's not an `Example` (and therefore seemingly not necessary to `Qed`) is the following in `noaliasBT.v`:
```coq
Lemma noaliasR2 s x y (f : Scan.form s) (g : NoAlias2.form x y s) :
               def f -> NoAlias2.eq_of g.
Proof. admit. Admitted.
```
To me, this lemma doesn't look provable, and its only use is in throwaway examples.

**Comment by anton-trunov** [comment]
> does it look provable to you?

It is provable, please see #17.

**Comment by anton-trunov** [comment]
Fixed by b9a21a0be531f7310152efa49a6445bc26746ae9
